wp-content/themes/core/assets/css/src/admin/acf/all.pcss

```diff
@@ -0,0 +1,37 @@
+/* Image Select Field Mods */
+@import "acf-image-select.pcss";
+
+/* Image Field Mods */
+@import "acf-image.pcss";
+
+/* Repeater Field Mods */
+@import "acf-repeater.pcss";
```

Comment:
Rather than both importing here and adding styles here, please keep the style selectors in a separate file. I might suggest naming this file `acf.pcss` and then importing it along with the other partials directly in `admin/block-editor/all.pcss`.